### PR TITLE
[SKU creator] fix wrong speed in case breakout of 2x50

### PR DESCRIPTION
**Pull request by madhanmellanox**
**- What I did**
I fixed the port split feature of SKU creator tool.

**- How I did it**
When trying to use port split feature of SKU creator tool to split a port in port_config.ini and config_db.json. It was not working, I fixed.

**- How to verify it**
With this fix, we should be able to split and unsplit a port in port_config.ini and config_db.json

**- Previous command output (if the output of a command-line utility has changed)**
No changes to CLI commands

**- New command output (if the output of a command-line utility has changed)**
No changes to CLI commands

### CODE REVIEW

**Comment by madhanmellanox** [comment]
retest please

**Review by liat-grozovik — DISMISSED** [review]

**Comment by liat-grozovik** [comment]
retest this please

File: scripts/sonic_sku_create.py [hunk 1]
```diff
@@ -56,7 +56,7 @@
     "1x1":   { "lanes":4, "speed":1000,   "step":4, "bko":0, "name": "etp" },
     "4x10":  { "lanes":4, "speed":10000,  "step":1, "bko":1, "name": "etp" },
     "4x25":  { "lanes":4, "speed":25000,  "step":1, "bko":1, "name": "etp" },
-    "2x50":  { "lanes":4, "speed":25000,  "step":2, "bko":1, "name": "etp" },
+    "2x50":  { "lanes":4, "speed":50000,  "step":2, "bko":1, "name": "etp" },
```
Comment (qiluo-msft) [inline scripts/sonic_sku_create.py:59]: >50000 [](start = 34, length = 5)

You mentioned "When trying to use port split feature of SKU creator tool to split a port in port_config.ini and config_db.json. It was not working".
I am wondering if this bug is not caught in unit test, and if there is a gap in unit test?

Comment (madhanmellanox) [reply scripts/sonic_sku_create.py:59]: The unit test in sonic-utilities for sonic_sku_create tool does not cover testing individual sub features. This is some how missed in manual testing and later when I tried to test it for documentation purposes, I found this issue. And, so addressed it.

Comment (qiluo-msft) [reply scripts/sonic_sku_create.py:59]: I see. Could you add more unit testcases to cover the changes here?

---
In reply to: [568833823](https://github.com/Azure/sonic-utilities/pull/1380#discussion_r568833823) [](ancestors = 568833823)

Comment (madhanmellanox) [reply scripts/sonic_sku_create.py:59]: Adding more unit test cases adds more changes to the sonic_sku_create.py script itself, that is why we had a basic unit test case to cover the main feature of the tool was written. But,  the tool itself is well tested manually now.

File: scripts/sonic_sku_create.py [hunk 2]
```diff
@@ -254,6 +250,9 @@ def json_file_parser(self, json_file):
         f_out = open(new_file, 'w')
         header_str = "#name           lanes                alias       index     speed\n"
         f_out.write(header_str)
+      
+        #data['PORT'] is already an OrderedDict, we can not sort it, so we create
```
Comment (qiluo-msft) [inline scripts/sonic_sku_create.py:254]: ># [](start = 8, length = 1)

Add one blank after `#`

File: scripts/sonic_sku_create.py [hunk 3]
```diff
@@ -254,6 +250,9 @@ def json_file_parser(self, json_file):
...
+        #a pi_list - list of port info items and then sort it
```
Comment (qiluo-msft) [inline scripts/sonic_sku_create.py:255]: >a [](start = 9, length = 1)

Is `a` intended word?

File: scripts/sonic_sku_create.py (hunk 2 above)
Comment (madhanmellanox) [reply scripts/sonic_sku_create.py]: addressed in next commit.

File: scripts/sonic_sku_create.py (hunk 3 above)
Comment (madhanmellanox) [reply scripts/sonic_sku_create.py]: addressed in next commit.

**Review by qiluo-msft — APPROVED** [review]

**Pull request merged**